src/examples/list/list-badge-icons.tsx

```diff
@@ -12,34 +12,34 @@ export class BadgeIconsListExample {
             text: 'King of Tokyo',
             secondaryText: '2-6 players',
             value: 1,
-            icon: 'gorilla',
+            icon: 'add_shopping_cart',
```

Comment:
Haha :smile: 
I understand we cannot redistribute the icons, but we should be able to use them within our own app right? :thinking:

Reply:
I mean even in the documentaion, the actual webclient

Reply:
Yes, we'll use them in the webclient. They're available as a private package from our internal Verdaccio. That seemed to break Greenkeeper though (although, that *could* be because Jenkins has been making trouble lately), and it doesn't feel great to depend on non-open source icons in the docs for an open source project.

But yes, when actually using `limel-icon`, any icons served from the correct path by the web server will be available to use.

Reply:
One idea I've toyed with is if we could switch from the public icon-set to our private one when building the docs for publishing. That way we could still get nice icons in the published docs…

Reply:
Yeah that was what I meant, we should still be able to have any icons we want in the docs, since we are only using them and not redistributing them